static/templates/settings/alert_word_settings.hbs

```diff
@@ -27,5 +27,14 @@
             </div>
         </div>
     </form>
-    <ul id="alert_words_list"></ul>
+    <div class="progressive-table-wrapper" data-simplebar>
+        <table class="table table-condensed table-striped wrapped-table">
+            <thead>
+                <th class="active" data-sort="alphabetic" data-sort-prop="full_name">{{t "Word" }}</th>
+                <th class="user_role" data-sort="role">{{t "Delete" }}</th>
```

Comment:
I think the `data-sort-prop` field here is copied from another table, and results in a sort indicator that doesn't work.